Added the Juggler Archetype

spthiessen opened the pull request:
Added the Bard archetype, Juggler.

spthiessen committed:
Add the Juggler pdf

spthiessen committed:
Added the Juggler to the json file

spthiessen committed:
Letter to A4

Changed the paper size of the Juggling Bard to A4 from Letter so it matches the rest of the project.

spthiessen committed:
Juggler Archetype

Created a new Juggler pdf file to reflect the new formatting that now conforms to the standard formatting ion this project.

spthiessen committed:
Old Juggler File

Deleted the old Juggler pdf that is no longer needed.

spthiessen committed:
Juggler Archetype Update

Here is the pdf for the Juggler archetype of the Bard so the class sheet does not overlap the copyright statement.

marcus-downing commented:
Just so you know, I have a script that exports all of the Illustrator files in the `pages` repo to produce the output in the `assets` repo - in all the different languages etc.

I'm at work, so I can't do a proper review until later, but this looks broadly good. Some minor points:

 - I prefer the base class name underneath rather than alongside the archetype name.
 - Since I've put Concentration as a pseudo-skill, at some point I'm going to remove Concentration (as well as other pseudo-skills like Disarm Traps etc) from the class pages.
 - There's some wasted space under the Arcane Spell Failure box. You can use the _distribute vertical space_ tool in Illustrator to share that space out among the boxes below.
 - In cases where a class "Gains X as a bonus feat", I prefer to at least summarise the rules of the feat if possible, rather than just referring them to docs elsewhere. It'll never cover 100% of what you need to know, of course, but it can help reduce misunderstandings.

spthiessen committed:
Formatting the Juggler

Changed the formatting for the Juggler archetype for the Bard to match the rest of the file

marcus-downing commented:
The JSON looks correct now.
